package.json

```diff
@@ -47,6 +47,7 @@
   },
   "workspaces": [
     "example",
+    "examples/example-wasm",
```

Comment:
Since the app is added as a workspace I don't expect `examples/example-wasm/package-lock.json` to be added as well.

Reply:
I'll remove the lock file 👍